cetmix_tower_yaml/tests/test_server_key.py

```diff
@@ -0,0 +1,16 @@
+from odoo.tests import TransactionCase
+
+
+class TestTowerKey(TransactionCase):
+    def setUp(self, *args, **kwargs):
+        super().setUp(*args, **kwargs)
+
+        self.Key = self.env["cx.tower.key"]
+
+    def test_key_secret_value_mask(self):
+        """Test that secret value is masked in YAML"""
+        key = self.Key.create(
+            {"name": "Test Key", "key_type": "s", "secret_value": "test_secret"}
```

Comment:
For better readability and adherence to Odoo coding style standards, it's recommended to format dictionaries with long keys and values in a column layout. This makes the code easier to read and maintain. For example:

key = self.Key.create({
    "name": "Test Key",
    "key_type": "s",
    "secret_value": "test_secret"
})

This format is helpful when adding new keys or modifying existing values.

Reply:
I agree, however this code was automatically formatted using pre-commit scripts.